repos/system_upgrade/common/actors/verifytargetversion/libraries/verifytargetversion.py

```diff
@@ -0,0 +1,54 @@
+from leapp import reporting
+from leapp.cli.commands import command_utils
+from leapp.exceptions import StopActorExecution
+from leapp.libraries.common.config import get_env
+from leapp.libraries.common.config.version import get_target_version
+from leapp.libraries.stdlib import api
+
+FMT_LIST_SEPARATOR = '\n    - '
+
+
+def process():
+    flavour = get_env('LEAPP_UPGRADE_PATH_FLAVOUR')
+    target_version = get_target_version()
+    supported_target_versions = command_utils.get_supported_target_versions(flavour)
+
+    leapp_unsupported = get_env('LEAPP_UNSUPPORTED', '0') == '1'
+    unsupported = target_version not in supported_target_versions
```

Comment:
Gets confusing having `leapp_unsupported` and `unsupported`, probably needs a better naming.
```suggestion
    is_target_version_unsupported = target_version not in supported_target_versions
```
or
```suggestion
    is_target_unsupported = target_version not in supported_target_versions
```

Reply:
refactored completely